spec/menu_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "menu"
+
+describe Menu do
+
+  subject { described_class.new }
+  
+  let(:menu) { { test_dish1: 7.50, test_dish2: 6.00, test_dish3: 4.00 } }
+  
+  before(:example) do
+    allow(subject).to receive(:menu).and_return(menu)
```

Comment:
Stubbing the class under test is never a good idea. You could use dependency injection here.. But then I'm wondering what you want from the class.